Makefile-test.mk

```diff
@@ -62,13 +62,15 @@ JOBSET_VERSION = $(shell $(GO_CMD) list -m -f "{{.Version}}" sigs.k8s.io/jobset)
 .PHONY: test
 test: gotestsum ## Run tests.
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverpkg=./... -coverprofile $(ARTIFACTS)/cover.out
+	cd cmd/experimental/kjobctl && make test
```

Comment:
we should add one maybe some dedicated ci jobs for kjobctl, I have opened https://github.com/kubernetes/test-infra/pull/32807 for this

Reply:
On this case we also need `verify` and `test-integration`.

Reply:
I hope we can put them all in one job

Reply:
`pull-kueue-test-kjobctl` is running `make test` in kjobctl dir, we should cange the kjobctl Makefile to run all that we want to be in CI

Reply:
Done